spec/i18n_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "i18n/tasks"
+
+describe "I18n sanity" do
+  let(:locales) do
+    ENV["ENFORCED_LOCALES"].presence || "en"
```

Comment:
What is the current config on the CI? 

We should do en, fr

Reply:
At the moment there is no configuration on the CI for the language, but I can add a configuration and update locales to `en`, `fr` in `spec/spec_helper.rb`. What do you think about this suggestion ?

Reply:
Perfect